StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/PayWithLinkController.swift

```diff
@@ -64,7 +64,7 @@ extension PayWithLinkController: PayWithLinkWebControllerDelegate {
             elementsSession: elementsSession,
             paymentOption: paymentOption,
             paymentHandler: paymentHandler,
-            isFlowController: false
+            integrationShape: .complete
```

Comment:
Maybe instead of passing in `integrationShape`, we just pass in the value for requiresInterstitialForCVC?

Reply:
We did something similar to integration shape in PaymentSheetLoader, and I tend to prefer this over the flag as it's more extensible for possible future changes in confirmation depending on surface. The flag `isFlowController` is a good example of why it's limiting IMO.